float32 Vehicle_speed 			#车速 km/h
float32 Motor_temperature 		#电机温度， 显示时带 ℃
float32 Controller_temperature          #控制器温度， 显示时带 ℃
uint8 State 				# 1--直线加速 2--八字绕环 3--高速循迹 4--EBS 5--车检 6--操控性
uint8 SOC 				#电池soc 0－100 ，显示时带％
float32 Battery_temp			#电池温度，显示时带℃
float32 Output_voltage                  #输出电压, 显示时带V
